src/sbf.cpp

```diff
@@ -1,590 +1,629 @@
-/****************************************************************************
```

Comment:
Can you avoid changing file permissions and from what it looks like, line endings, and whitespace?

Reply:
The commands get stuck in a loop, Mirko Denecke tested it as well and he got the same 

[TXT of loop](https://fromsmash.com/SBF-Heading-config-loop)